src/DataTypes/TimezoneMixin.h

```diff
@@ -15,7 +15,7 @@ class TimezoneMixin
 
     explicit TimezoneMixin(const String & time_zone_name = "")
         : has_explicit_time_zone(!time_zone_name.empty())
-        , time_zone(DateLUT::instance(time_zone_name))
+        , time_zone(time_zone_name.empty() ? DateLUT::instance() : DateLUT::instance(time_zone_name))
```

Comment:
Why?

Reply:
I would rather add `DateLUT::serverTimezone()` function and use it explicitly from corresponding places. Also, is `TimeZoneMixin` used inside logs? I thought that logging and other lower-level things (like TTL) it use `DateLUT` directly...